### ISSUE TITLE
Figures seems to be missing

### ISSUE THREAD

**Issue by talgalili** [hadley/adv-r#816]
For example here:
http://adv-r.had.co.nz/memory.html
Something seems off with the image URL.

**Comment by justbennet** [comment]
This issue was first reported in Issue #812 and there are two missing figures, the URLs for which are

    http://adv-r.had.co.nz/temp_files/figure-html/size-q-1.png
    http://adv-r.had.co.nz/temp_files/figure-html/size-a-1.png

The relevant lines  from memory.tex are lines 101 and 191, shown here with context.

```
./memory.tex-99-\end{Shaded}
./memory.tex-100-
./memory.tex:101:\includegraphics{figures/size-q-1.pdf}
./memory.tex-102-
./memory.tex-103-This isn't just an artefact of integer vectors. Every length 0 vector
--
--
./memory.tex-189-\end{Shaded}
./memory.tex-190-
./memory.tex:191:\includegraphics{figures/size-a-1.pdf}
./memory.tex-192-
./memory.tex-193-Beyond 128 bytes, it no longer makes sense for R to manage vectors.
```

Those `\includegraphics{}` calls refer to a nonexistent directory.

**Comment by hadley** [comment]
Will fix as part of #798
